docs/News.md

```diff
@@ -1,3 +1,49 @@
+## July 29, 2024
+*   **Added data** consisting of 12,863 samples from 33 TCGA and 10 CPTAC studies from the [Genomic Data Commons (GDC)](https://gdc.cancer.gov) as part of the [Cancer Research Data Commons (NCI-CRDC)](https://datacommons.cancer.gov) initiative. The studies are generated through the [ISB-CGC BigQuery tables](https://bq-search.isb-cgc.org/search?status=current).
```

Comment:
Can we change the last sentence to "More information can be found on our FAQ" and link to this page
https://github.com/cBioPortal/cbioportal/blob/699e939482228c55321afad3fd9ee9c237949cf9/docs/user-guide/faq.md#tcga

Reply:
Thanks @n1zea144! Updated.